Delicate-Butterflies/Bangazon-Site
# Add product to cart

Related Issue: #40 

Description: User can add any product that they view, to the cart.

PR Testing Steps:
- npm start
- product-type -> product -> view
- should ask you to login if not logged in
- click add to cart
- should go to shopping cart and it should be there.
- complete all current orders.
- add product again and it should create new order with that product.

Check list before you make a PR:
